src/api/v1/index.ts

```diff
@@ -10,35 +10,51 @@
  * - api.json
  */
 import * as Hapi from 'hapi';
+import * as AuthJwt from 'hapi-auth-jwt2';
 import organisations from './organisations';
 import surveys from './surveys';
 import constants from './constants';
 import validateUser from '../../auth/scheme/validate_user';
 import validateExternal from '../../auth/scheme/validate_external';
-import * as AuthJwt from 'hapi-auth-jwt2';
+import addLifecycleHooks from './hooks';
 const AuthBearer = require('hapi-auth-bearer-token');
 
 export default {
   name: 'Twine API v1',
   register: async (server: Hapi.Server, options: { jwtSecret: string }) => {
-    await server.register([{
-      plugin: AuthJwt,
-      once: true },
-    {
-      plugin: AuthBearer,
-      once: true }]
-    );
+    /*
+     * Authentication Strategies
+     */
+    await server.register([
```

Comment:
always hating on my layout...

Reply:
lol, can't have them inline when there's 20 of them